docs/getting-started/quick-start-guides/deploy-rancher-manager/helm-cli.md

```diff
@@ -15,14 +15,14 @@ The full installation requirements are [here](../../../pages-for-subheaders/inst
 
 ## Install K3s on Linux
 
+Rancher needs to be installed on a supported Kubernetes version. To specify the K3s version, use the INSTALL_K3S_VERSION environment variable when running the K3s installation script. Refer to the [support maintenance terms](https://rancher.com/support-maintenance-terms/).
+
 Install a K3s cluster by running this command on the Linux machine:
 
 ```
-curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION="***" sh -s - server --cluster-init
+curl -sfL https://get.k3s.io | INSTALL_K3S_VERSION="v1.24.11+k3s1" sh -s - server --cluster-init
```

Comment:
the K3S version is left blank to be generic, as the version to use might change depending on various requirements

Reply:
this is right, but it makes hard to figure out which format should it be